LICENSE.md

```diff
@@ -1,4 +1,4 @@
-# <div align="center">«NexT» – Elegant and powerful theme for Hexo.</div>
```

Comment:
Is this already en dash?

Reply:
Subtitle uses en dash now: `1v9's Blog – The magic ends here`, not hyphen.

Reply:
No, it's hyphen. `"–"=="-"` returns false, you can have a try in JS console.